internal/pkg/meta/constants.go

```diff
@@ -0,0 +1,16 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package meta
+
+const (
+	// Upgrade is the upgrade tag.
+	Upgrade = iota + 6
```

Comment:
what's the significance of the number `6`, or just that the lower numbers are used anywhere else?

Reply:
it came as I split the core ADV code and the constants used in Talos itself.

this way the constants keep same values as before, but we don't have constants which make zero sense for Talos (inherited from syslinux days)